src/uniprotkb/config/evidenceCodes.ts

```diff
@@ -91,126 +93,129 @@ const manualLabelRenderer = (evidences: Evidence[]) => {
 export const ecoCodeToData = {
   [ecoCode.EXP]: {
     manual: true,
-    label: 'Manual assertion based on experiment',
+    label: () => 'Manual assertion based on experiment',
     description: 'Inferred from experiment',
     labelRender: publicationCountRenderer,
```

Comment:
Feel like there's a naming issue here between these 3 fields that we'll stumble on in the future.
Should we find a way to rename this throughout the codebase to something clearer? I understand that we use some of them as fallback for the other in some places, so it might still not make it obvious.
I can see the `labelRender` is is used for the evidence tag label, and `label` for the evidence tag content heading. Should we call them like that? (`evidenceTagLabel` & `evidenceTagContentHeading`)
And `description` is used for the GO section, right? should that be `evidenceTagContentHeadingForGO`